spec/models/user_spec.rb

```diff
@@ -1,9 +1,26 @@
 require 'spec_helper'
 
 describe User do
-  describe "#generate_unsubscribe_token" do
-    subject { create(:user) }
+  subject { create(:user) }
+
+  describe "attributes" do
+    it { should respond_to(:bridge_troll_user_id) }
+    it { should respond_to(:rails_bridge_class_count) }
+  end
 
+  describe ".rails_bridge_users" do
+    context "when some users have given their bridge troll id" do
+      before do
+        2.times { FactoryGirl.create(:user, bridge_troll_user_id: "") }
+        2.times { FactoryGirl.create(:user, bridge_troll_user_id: "foo") }
```

Comment:
No need to use the `FactoryGirl` namespace here, just `create` will suffice.

cf. https://github.com/rails-school/school/blob/10c272cd312bfaaf08538aa4a7915deb2bb7c897/spec/support/factory_girl.rb#L2-Lundefined